Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN wget "https://downloads.apache.org/spark/spark-${SPARK_VERSION}/spark-${SPAR
 WORKDIR /home/app/frontend/
 RUN yarn install && yarn build
 
-FROM openjdk:11-jre-slim-bullseye
+FROM eclipse-temurin:11-jdk-jammy
```

Comment:
Previous image was of JRE, maybe it would work with `eclipse-temurin:11-jre-jammy`?

Reply:
I also assumed that initially.
However, if you use JRE, the stage running `gradle` throw an exception saying that it cannot find java compiler

Reply:
You are right. But in that case we could do the compiling in JDK and for runtime image use JRE here.

Reply:
ok I'll have a try

Reply:
and it works and is included in the latest commit